### ISSUE TITLE
ios simulator pinch_zoom_image 

### ISSUE THREAD

**Issue by hosseinbahmanabadi** [YoussefKababe/pinch_zoom_image#5]
=== BUILD TARGET sqflite OF PROJECT Pods WITH CONFIGURATION Debug ===
    /flutter/flutter/.pub-cache/hosted/pub.dartlang.org/pinch_zoom_image-0.2.5/ios/Classes/SwiftPinchZoomImagePlugin.swift:13:55: error: 'UIWindowLevelStatusBar' has been renamed to 'UIWindow.Level.statusBar'
            UIApplication.shared.keyWindow?.windowLevel = UIWindowLevelStatusBar
                                                          ^~~~~~~~~~~~~~~~~~~~~~
                                                          UIWindow.Level.statusBar
    UIKit.UIWindowLevelStatusBar:2:12: note: 'UIWindowLevelStatusBar' was obsoleted in Swift 3
    public let UIWindowLevelStatusBar: UIWindow.Level
               ^
    /flutter/flutter/.pub-cache/hosted/pub.dartlang.org/pinch_zoom_image-0.2.5/ios/Classes/SwiftPinchZoomImagePlugin.swift:16:55: error: 'UIWindowLevelNormal' has been renamed to 'UIWindow.Level.normal'
            UIApplication.shared.keyWindow?.windowLevel = UIWindowLevelNormal
                                                          ^~~~~~~~~~~~~~~~~~~
                                                          UIWindow.Level.normal
    UIKit.UIWindowLevelNormal:2:12: note: 'UIWindowLevelNormal' was obsoleted in Swift 3
    public let UIWindowLevelNormal: UIWindow.Level

**Comment by sk3llo** [comment]
Having the same issue with my iOS simulator

**Comment by sh0umik** [comment]
Any Solution yet , I think the swift version is outdated

**Comment by dwlrathod** [comment]
Yes, the swift version is outdated. No more working in iOS device.

**Comment by furkankurt** [comment]
I created new repo. Owner of this project don't accept pull requests. You can use mine.
https://pub.dartlang.org/packages/pinch_zoom_image_last
If there is an issue, you can open an issue of my github repo.
